curated_transformers/tokenization/roberta_tokenizer.py

```diff
@@ -115,6 +117,13 @@ def from_files(
             eos_piece=eos_piece,
         )
 
+    @property
+    def eos_piece(self) -> Optional[str]:
+        return self._eos_piece
+
+    def piece_to_id(self, piece: str) -> Optional[int]:
+        return self.processor.token_to_id(piece)
```

Comment:
Same as above but to `ByteBPETokenizer`.

Reply:
Moved.